content/blog/2016/2016-06-02-save-costs-with-ec2-spot-fleet.adoc

```diff
@@ -0,0 +1,67 @@
+
+In this walkthrough, we’ll show you how to configure Jenkins to automatically scale a fleet of Spot instances up or down depending on the number jobs to be completed. 
+
+Request an Amazon EC2 Spot fleet
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+To get started, login to https://console.aws.amazon.com/ec2/[Amazon EC2 console^], and click on *Spot Requests* in the left hand navigation pane. Alternatively, you can directly login to https://console.aws.amazon.com/ec2sp/v1/spot[Amazon EC2 Spot Requests console^]. Then click on the *Request Spot Instances* button at the top of the dashboard. 
+
+In the Spot instance launch wizard, select the *Request & Maintain* option to request a Spot fleet that automatically provisions the most cost-effective EC2 Spot instances, and replenishes them if interrupted. Enter an initial target capacity, choose an AMI, and select multiple instance types to automatically provision the lowest priced instances available.
+
+image:https://i.imgur.com/wDW63sm.jpg[alt=""]
+
+On the next page, ensure that you have selected a key pair, complete the launch wizard, and note the Spot fleet request ID. 
+
+Amazon EC2 Spot fleet automates finding the lowest priced instances for you, and enables your Jenkins cluster to maintain the required capacity; so, you don’t need any bidding algorithms to provision the optimal Spot instances over time. Note that as long as you bid higher than the current Spot market price, you will always pay only the market price for your Spot instances, which is on average 80-90% lower than regular EC2 On-demand prices. The https://console.aws.amazon.com/ec2sp/v1/spot[Amazon EC2 Spot Requests console^] enables you automate these bids through a Spot fleet request. You can also download the JSON configuration from the console to use with the AWS CLI and SDK toolkit.
+
+Configure Jenkins
+~~~~~~~~~~~~~~~~~
+Install the Plugin
+^^^^^^^^^^^^^^^^^^
+From the Jenkins dashboard, select *Manage Jenkins*, and then click *Manage Plugins*. On the *Available* tab, search for and select the _EC2 Fleet Jenkins Plugin_. Then click the *Install* button. 
+
+image:https://i.imgur.com/iOfvSD8.jpg[alt="", width="600",height="180"]
+
+After the plugin installation is completed, select *Manage Jenkins* from the Jenkins dashboard, and click *Configure System*. In the *Cloud* section, select _Amazon Spot Fleet_ to add a new Cloud. 
+
+image:https://i.imgur.com/MtvaRLU.jpg[alt=""]

```

Comment:
I suggest creating a directory in `/content/images/post-images/` to contain all the images.